javascript/README.md

```diff
@@ -63,6 +63,21 @@ console.log(ele.outerHTML);
 // <p class="budou-this" style="word-break: keep-all; overflow-wrap: break-word;">今日は<b><wbr>とても<wbr>天気</b>です。</p>
 ```
 
+There is another way to apply the process to an HTML element.
+
+```javascript
```

Comment:
Let's add the line `import { Applier } from 'budoux';` to show how users can import the Applier class.

Reply:
Done, thanks for the good suggestion.